testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -26,6 +26,8 @@
     SUPPORTS_LIMIT_PUSHDOWN,
 
     SUPPORTS_TOPN_PUSHDOWN,
+    // TODO: Remove this once https://github.com/trinodb/trino/issues/7496 is done
+    SUPPORTS_TOPN_PUSHDOWN_WITH_VARCHAR(SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY),
```

Comment:
this should default on a conjunction of SUPPORTS_TOPN_PUSHDOWN and SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY